### DISCUSSION TITLE
add --version option

### DISCUSSION THREAD

**Discussion by c33s** [samuel-lucas6/Kryptor#60] [Ideas]
```
kryptor --version
version: 4.0.1
```

**Comment by samuel-lucas6** [comment]
There's currently `-a|--about` for the version and license. Do you think there should be `--version` as well as or instead of that?

I have thought about this before. The only reason it doesn't exist is because I was trying to give every option a one letter equivalent and `-v` is taken for `--verify`. `--version` would be the first exception because there's no way of renaming `--verify`.

**Comment by c33s** [comment]
it is kind of convention that tools have a `--version` and a `--help` flag. it feels kind of confusing if they are missing. your concern about the one letter equivalent is valid, i would simply choose to not have a one letter flag for version or go with the uppercase letter `-V`.

**Reply by samuel-lucas6** [reply]
Yeah, that's fair. I was trying to avoid uppercase letters as well because I think it becomes confusing, but I will swap `-a|--about` for `--version`.
